dht-node/src/dht_node/index.ts

```diff
@@ -29,14 +24,20 @@ export type CommunityApi = {
   url: string
 }
 
+type KeyPair = { publicKey: Buffer; secretKey: Buffer }
+
 export const startDHT = async (topic: string): Promise<void> => {
   try {
     const TOPIC = DHT.hash(Buffer.from(topic))
-    const keyPair = DHT.keyPair(getSeed())
+    const keyPair = DHT.keyPair(
```

Comment:
missing the comment like in other PRs
it's a pain to review several times the same code with the same remarks